examples/theater-metadata-instance.ttl

```diff
@@ -0,0 +1,319 @@
+@prefix vann:                  <http://purl.org/vocab/vann/> .
+@prefix schema:                <https://schema.org/> .
+@prefix ids:                   <https://w3id.org/idsa/core/> .
+@prefix dcat:                  <http://www.w3.org/ns/dcat#> .
+@prefix voaf:                  <http://purl.org/vocommons/voaf#> .
+@prefix wikidata:              <https://www.wikidata.org/entity/> .
+@prefix gnd-kg:                <https://d-nb.info/gnd/> .
+@prefix dbr:                   <http://dbpedia.org/resource/> .
+@prefix lcc-3166-1:            <https://www.omg.org/spec/LCC/Countries/ISO3166-1-CountryCodes/> .
+@prefix lcc-3166-2-de:         <https://www.omg.org/spec/LCC/Countries/Regions/ISO3166-2-SubdivisionCodes-DE/> .
+@prefix lang:                  <https://w3id.org/drk/vocabs/languages-iso639-1#> .
+@prefix tgenres:               <http://w3id.org/drk/vocabs/theatrical-genres#> .
+@prefix tprod:                 <http://w3id.org/drk/vocabs/theatrical-production-types#> .
+@prefix tchar:                 <http://w3id.org/drk/vocabs/theatrical-character-types#> .
+@prefix ppfa:                  <http://w3id.org/drk/vocabs/personal-profile-for-accessibility#> .
+@prefix tevent:                <http://w3id.org/drk/vocabs/theatrical-event-types#> .
+@prefix fibo-fnd-acc-4217:     <https://spec.edmcouncil.org/fibo/ontology/FND/Accounting/ISO4217-CurrencyCodes/> .
+
+# Note: This is just an example and it may not be factually correct.
+
+####################################################################
+# Instance of schema:TheaterGroup (subclass of schema:Organization).
+# Theater (Organization) naming scheme used here: <ISO3166-1 Country code>_<City name>_<(Shortened) Theater name>.
+drkr:DE_Augsburg_STAugsburg
+    a              schema:TheaterGroup ;
+    schema:name    "Staatstheater Augsburg"@en, "Staatstheater Augsburg"@de ;
+    schema:description
+                   "Staatstheater (State Theater) Augsburg is a prominent German theater organization in Augsburg, offering a diverse range of performances including opera, theater, plays, ballet, and concerts."@en ;
+    schema:description
+                   "Das Staatstheater Augsburg ist ein bedeutendes deutsches Theater in Augsburg, das ein vielfältiges Angebot an Aufführungen wie Oper, Theater, Schauspiel, Ballett und Konzerte bietet."@de ;
+    schema:address [ a                      schema:PostalAddress ;
+                     schema:name            "Staatstheater Augsburg - Front Desk"@en ;
+                     schema:name            "Staatstheater Augsburg - Pforte"@de ;
+                     schema:streetAddress   "Provinostr. 52"@en ;
+                     schema:postalCode      "86153" ;
+                     schema:addressLocality "Augsburg"@en ;
+                     schema:addressRegion   lcc-3166-2-de:Bayern ;
+                     schema:addressCountry  lcc-3166-1:Germany ;
+                     schema:url             <https://maps.app.goo.gl/C9oHeBUtVhWYFwX68> ] ;
```

Comment:
For pointing to a service as specific as Google Maps, does it make sense to use a property as general as `schema:url`?

Reply:
This indeed needs to be updated.

`schema:Place` has a `hasMap` property, however it's meant to specify a URL to a map of the place (such as a ParkingMap, SeatingMap, TransitMap, or a VenueMap). So we can't use it for this purpose.

One potential solution is to create a new property:

```turtle
drk:mapLocationURL
    a                  rdf:Property, owl:DatatypeProperty ;
    rdfs:subPropertyOf schema:url ;
    rdfs:domain        schema:Place ;
    rdfs:range         schema:URL ;
    rdfs:label         "Map Location URL"@en ;
    rdfs:comment       "A URL linking to the geographical location of the specified place on a digital map service for navigation purposes (e.g., Google Maps, OpenStreetMap, Apple Maps). This differs from a venue map or seating chart by offering a direct navigation point, such as <https://maps.app.goo.gl/v1XhJ3uVdixjGgY2A>."@en .
```

@clange, please let me know if you have any other/further suggestions.